stitch/src/main/java/com/mongodb/stitch/android/auth/AvailableAuthProviders.java

```diff
@@ -16,21 +21,25 @@
     private final GoogleAuthProviderInfo _google;
     private final AnonymousAuthProviderInfo _anon;
     private final EmailPasswordAuthProviderInfo _emailPass;
+    @NonNull
+    private final List<CustomAuthProviderInfo> _customAuths;
```

Comment:
[q] Why is this a list?

Reply:
I was under the impression that we are going to allow multiple Custom Auth Providers?

Reply:
We may eventually for all types, but not now. I'd use a single one for now.